src/com/company/Menu.java

```diff
@@ -0,0 +1,37 @@
+package com.company;
+
+public class Menu {
+    Contact [] listContacts;
+    int position;
+
+    Menu(){
+        position = 0;
+        listContacts = new Contact[500];
+    }
+
+    public void AddContact(Contact newUser){
+       for (int i = 0; i < position; i++) {
+           listContacts[position] = newUser;
+           position++;
+       }
+       System.out.println("User added successfully"+ listContacts[position]);
+
+    }
+
+    public void ShowContact(){
+        int index=0;
+        do{
+            System.out.println(listContacts[index]);
+            index++;
+        }while(listContacts[index].name.equals("")!= false);
```

Comment:
@SFDC-Melby-Arguedas Getter for name attribute could be used in this part:
_listContacts[index].getName().equals("")_